### DISCUSSION TITLE
Visualize and Finalize the Result

### DISCUSSION THREAD

**Discussion by HocheolJang426** [aertslab/SCENIC#280] [R] [answered]
Hi 

I'm wondering how to sort and finalize the result.
in output folder, MotifEnrichment_preview.html made, but this file is too complicated
NES are not arranged, and many of TF_highConf overlapped
I also checked int folder, motifEnrichment_selfMotifs_wGenes.Rds, also this data.frame is complicated

I want to export my result like;
motif logo ; NES ; TF name ; Target gene
or any other good way to visualize the SCENIC result/output

Is there a good way to visualize the result?

**Comment by s-aibar** [comment, marked as answer]
Dear @HocheolJang426 ,

I think I don't fully understand your question. You can export and explore the motif enrichment with any format you prefer... The "MotifEnrichment_preview.html", is indeed just that, a preview than you can then re-shape to fit your needs.
If you already know which columns you would like to show, just load the `motifEnrichment_selfMotifs_wGenes.Rds` file in R, filter, re-shape, and export it with `DT::datatable()`...

**Reply by HocheolJang426** [reply]
My first question is...Is it possible to export motif logo in higher magnification?
I'am wondering that I can export or show the result like this;
![image](https://user-images.githubusercontent.com/70268227/162757970-ce24a18c-bd86-4576-8977-bdea2efe3a27.png)

**Reply by s-aibar** [reply]
The logo is provided as a fixed image, so there is no easy way to change the magnification (besides your own brower settings).

In [RcisTarget vignette](https://bioconductor.org/packages/release/bioc/vignettes/RcisTarget/inst/doc/RcisTarget_MainTutorial.html) you have an example on how to use the package [DT](https://rstudio.github.io/DT/) to show the motifs... I think that is mostly what you need (just select the appropriate columns, and [reshape](https://seananderson.ca/2013/10/19/reshape/) the TFs if needed...)


<img width="1006" alt="image" src="https://user-images.githubusercontent.com/20438544/162758866-04bed04e-b93b-487c-b311-02d57c3feaaf.png">

**Reply by HocheolJang426** [reply]
Thanks for your kindness, your answer really helped me

when I use function datatable(), logo disappear
But when I use viewMotif function in SCENIC, logo appear
I think I have to use viewMotif function, especially the output of SCENIC

I have one more question... last question
How can I export html image in R "viewer" menu?
viewMotif or datatable() function both show data in viewer menu in R studio
But I want to export this image like jpg or png(or tiff)
Is it possible?

**Reply by s-aibar** [reply]
I don't think you can save the table as image, but if I remember correctly, there is an option in the viewer to export as html (otherwise, check the DT manual, there is some way to do it programmatically).
